extensions/amp-ad/0.1/amp-ad-api-handler.js

```diff
@@ -106,14 +106,10 @@ export class AmpAdApiHandler {
           if (data.width !== undefined) {
             newWidth = Math.max(this.element_./*OK*/offsetWidth +
                 data.width - this.iframe_./*OK*/offsetWidth, data.width);
-            this.iframe_.width = newWidth;
```

Comment:
Wait, when we try to update size, we resize iframe immediately. But even the amp-ad element size could remain unchanged?